src/tailwind.preset.js

```diff
@@ -53,12 +55,23 @@ export default {
           light: '#737373',
         },
       },
+      ringColor: {
+        DEFAULT: '#59a7e8',
```

Comment:
Might be worth linking to the original PR from Kyle that introduced the custom ring styling to document the accessibility reasons why we customized it.

Reply:
Here's a PR in which the `focus-visible` polyfill was added to the client: https://github.com/hypothesis/client/pull/1479

Reply:
OK, I've timed out and can't find anything useful to add at the moment. The plugin explains what it does fairly clearly; we can add a "why" as a simple follow-up if we determine what we'd like to say :)